python_terraform/terraform.py

```diff
@@ -456,6 +456,25 @@ def show_workspace(self, **kwargs) -> CommandOutput:
         """
         return self.cmd("workspace", "show", **kwargs)
 
+    def list_workspace(self) -> List[str]:
+        """List of workspaces
+        
+        :return: workspaces
+        :example:
+            >>> tf = Terraform()
+            >>> tf.list_workspace()
+            ['default', 'test']
+        """
+        return list(
+            filter(
+                lambda x: len(x) > 0,
+                map(
+                    lambda x: x.strip('*').strip(),
```

Comment:
sorry, one more NIT item, could you rename the `x` , and give it a meaningful name?